pkg/scanners/scannerv4/scannerv4.go

```diff
@@ -148,6 +153,11 @@ func (s *scannerv4) GetVulnDefinitionsInfo() (*v1.VulnDefinitionsInfo, error) {
 		return nil, fmt.Errorf("pulling metadata from matcher: %w", err)
 	}
 
+	lastTms := metadata.GetLastVulnerabilityUpdate()
+	if lastTms == nil || lastTms.Equal(protoTypes.Timestamp{}) || lastTms.Equal(zeroProtoTimestampFromTime) {
+		return nil, errors.New("no timestamp available")
```

Comment:
How does this look in the UI?

Reply:
Before:
![image](https://github.com/stackrox/stackrox/assets/119438707/141ff261-dce6-4a10-9d94-85aa7c0209aa)

After:
![image](https://github.com/stackrox/stackrox/assets/119438707/109575e4-a9ef-4aee-8442-4b03fe44a603)